ui/client/components/HealthCheck.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import {connect} from "react-redux"
+import ActionsUtils, {EspActionsProps} from "../actions/ActionsUtils"
+import InlinedSvgs from "../assets/icons/InlinedSvgs"
+import HttpService from "../http/HttpService"
+import PeriodicallyReloadingComponent from "./PeriodicallyReloadingComponent"
+import {withTranslation} from "react-i18next"
+import {WithTranslation} from "react-i18next/src"
+import {compose} from "redux"
+
+type HealthCheckResponse = {
+  state: string,
+  error: string,
+}
+
+type OwnProps = {}
+
+type State = {
+  healthCheck?: HealthCheckResponse,
+}
+
+class HealthCheck extends PeriodicallyReloadingComponent<Props, State> {
+
+  public static STATE_OK = "ok"
+
+  onMount(): void {
+    //Empty implementation for abstract class
+  }
+
+  getIntervalTime = () => this.props.healthCheckInterval
+
+  reload = () => {
+    HttpService.fetchHealthCheck().then((check: HealthCheckResponse) => this.setState({healthCheck: check}))
+  }
+
+  render() {
+    const {t} = this.props
```

Comment:
t? Maybe more descriptive name?

Reply:
It's props i18next.

Reply:
Ok, but we can rename it https://stackoverflow.com/questions/52486091/how-to-rename-a-variable-coming-into-a-function-in-typescript - WDYAT?